src/sequence_batch_scheduler/sequence_batch_scheduler.cc

```diff
@@ -367,6 +367,14 @@ SequenceBatchScheduler::GenerateInitialStateData(
             std::to_string(initial_state.dims().size()) +
             " != " + std::to_string(state.dims().size()));
   }
+  const auto& initial_state_pair = initial_state_.emplace(
+      std::piecewise_construct, std::forward_as_tuple(state.input_name()),
+      std::forward_as_tuple(initial_state.name()));
+  auto& initial_state_data = initial_state_pair.first->second;
+
+  if (max_batch_size_ > 0) {
+    initial_state_data.shape_.emplace_back(1);
```

Comment:
Just sanity checking, for a request with batch size > 1, what is the expected behavior here?

Reply:
Sequence batcher doesn't allow requests with batch size larger than 1.

Reply:
That's what I figured, just double checking 🙏